itl/plugins-contrib.d/databases.conf

```diff
@@ -762,7 +762,7 @@ object CheckCommand "elasticsearch" {
 object CheckCommand "redis" {
 	import "ipv4-or-ipv6"
 
-	command = [ PluginContribDir + "/check_redis.pl" ]
+	command = [ PluginContribDir + "/check_redis" ]
```

Comment:
But it **is** called check_redis.pl, isn't it? https://github.com/willixix/naglio-plugins/blob/master/check_redis.pl

Reply:
Shit, nevermind, i close the merge request.
I don´t know why but I was very shure, this had to be in the monitoring-plugins repository